cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxEditBoxHelper.java

```diff
@@ -157,6 +158,15 @@ public void afterTextChanged(Editable s) {
 
                 editBox.setOnFocusChangeListener(new View.OnFocusChangeListener() {
 
+                	 private Handler mHandler = new Handler();
+                	 private Runnable decor_view_settings = new Runnable()
```

Comment:
the indentation is wrong here and the variable name `decor_view_settings` doesn't confirm to the coding style.
CameCase naming convention is used here.